app/models/settings/user_experience.rb

```diff
@@ -11,6 +11,7 @@ class UserExperience < Base
     # basic (current default), rich (cover image on all posts), compact (more minimal)
     setting :feed_style, type: :string, default: "basic"
     setting :home_feed_minimum_score, type: :integer, default: 0
+    setting :index_minimum_score, type: :integer, default: 0
```

Comment:
Should this default be 1 just to be safe? My concern is that if a Forem creator forgets to update this value, all of their posts would be indexed (probably a good thing) AND in the sitemap (probably not so great).

I don't have a great understanding of the SEO implications though. cc @benhalpern for his thoughts.